src/ui/buttons/button-animations/index.js

```diff
@@ -3,7 +3,18 @@
 import { type Personalization } from '../props';
 
 import { setupDivideLogoAnimation } from './divide-logo-animation';
-import { type ButtonAnimationOutputParams } from './types';
+import { setupFadeOutLogoAndShowLabelAnimation } from './fadeout-logo-show-label-text';
+import type { ButtonAnimationIds, ButtonAnimationOutputParams } from './types';
+
+
+function setupAnimation(animationId, animationLabelText) : Function | null {
+    const animationIds : ButtonAnimationIds = {
+        'run-divide-logo-animation':        setupDivideLogoAnimation,
+        'alternate-slide-logo-animation':   setupFadeOutLogoAndShowLabelAnimation
+    };
+
+    return (animationIds[animationId] && animationIds[animationId](animationLabelText)) || null;
```

Comment:
`|| null` is not necessary here since we are handling a falsy value in line 35

Reply:
I change that portion of code now it looks like this:
![Screen Shot 2021-11-01 at 2 05 20 PM](https://user-images.githubusercontent.com/7244087/139726887-3e29961a-ba2b-4459-8337-d67bb4512530.png)

Reply:
I agree with you guys, I removed the null value on line 16